deployment/k8s/charts/components/bootstrapK8s/templates/bootstrap-job.yaml

```diff
@@ -0,0 +1,100 @@
+{{- if not .Values.skipImageBuild }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: {{ .Values.namespace }}
+  name: bootstrap-k8s
+  labels:
+    app.kubernetes.io/component: bootstrap-k8s
+spec:
+  backoffLimit: 0
+  ttlSecondsAfterFinished: 3600  # keep for 1 hour
+  template:
+    spec:
+      serviceAccountName: {{ .Values.serviceAccountName }}
+      restartPolicy: Never
+      volumes:
+        {{- if .Values.localRepoMount }}
+        - name: local-mount
+          hostPath:
+            path: /opensearch-migrations
+            type: DirectoryOrCreate
+        {{- end }}
+      terminationGracePeriodSeconds: 0
+      {{- if .Values.awsEKSEnabled }}
+      nodeSelector:
+        eks.amazonaws.com/compute-type: auto
+      {{- end }}
+      containers:
+        - name: bootstrap-k8s
+          image: amazoncorretto:17-al2023
+          env:
+            - name: GRADLE_OPTS
+              value: "-Xms512m -Xmx4g"
+          command:
+            - sh
+            - -c
+          args:
+            - |
+              set +e
+              yum update -y && yum install -y git findutils tar unzip rsync docker vim && yum clean all
+
+              ARCH=$(uname -m)
+                case "$ARCH" in
+                aarch64)
+                  ARCH="arm64"
+                  AWS_CLI_URL="https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip"
+                  ;;
+                x86_64)
+                  ARCH="amd64"
+                  AWS_CLI_URL="https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip"
+                  ;;
+                *)
+                  echo "Unsupported architecture: $ARCH"
+                  exit 1
+                ;;
+              esac
+
+              echo "Installing AWS cli"
+              curl -sSL "$AWS_CLI_URL" -o awscliv2.zip
+              unzip awscliv2.zip
+              ./aws/install
+              rm -rf aws awscliv2.zip
+              aws --version
+
+              rm -rf /workspace/*
+              rm -rf /workspace/.??*
+              {{- if not .Values.localRepoMount }}
+              echo "Cloning repo from {{ .Values.repositoryUrl }} (branch: {{ .Values.repositoryBranch }})..."
+              git clone --branch {{ .Values.repositoryBranch }} {{ .Values.repositoryUrl }} /workspace
+              {{- else }}
+              rsync -av --exclude=build --exclude=cdk.out --exclude=node_modules --exclude=.gradle --exclude=.idea --exclude=.git /local-workspace/ /workspace/
+              {{- end }}
+              cd /workspace
+
+              docker buildx create --name local-remote-builder --driver remote tcp://buildkitd:1234
```

Comment:
nit: a comment stating that this connects to the buildkit pod deployed elsewhere in the chart would be helpful

Just curious, can buildkit with this configuration run parallel builds?